app/screens/settings/about/about.tsx

```diff
@@ -1,10 +1,13 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
+import Clipboard from '@react-native-clipboard/clipboard';
 import React, {useCallback, useMemo} from 'react';
 import {useIntl} from 'react-intl';
 import {Alert, Text, View} from 'react-native';
 import DeviceInfo from 'react-native-device-info';
+import {Button} from 'react-native-elements';
```

Comment:
Instead of using the button from `react-native-elements` it should be simpler to use the one on `@components/button`. That way most of the styling should be done by the component. I don't remember if it supports icons out of the box, but it would be nice to give it a try.

Reply:
willl check that! btw should I add testing for this .

Reply:
Hi @larkox, it does not support any Icons, except Mattermost logo

Reply:
Heyyy got ittttt, I will make the changes

Reply:
> Re: testing
> 
> It would be awesome if you add some testing. So far, we are not enforcing those for community contributions, but of course we love having them. We have both unit tests with jest and e2e tests on detox. Not sure what do you have in mind, but feel free to add them. If you have any doubt, do not hesitate to ask me.

Yeah, couple days ago I was trying to write the test 😄, but i Could not get any other tests to refer to so stopped it. 
And I have a couple of doubts, for writing tests what is prefered ? writing in the same folder or writing in the main test folder ?

Reply:
For e2e test, write them on the `detox/e2e/test` folder. Not on the root, but a relevant folder. For example... `detox/e2e/test/about`.

For unit tests, in the same folder. For example, if you have `screens/settings/about/about.tsx` the test that test that would be `screens/settings/about/about.test.ts(x)`.

If you look for any file called `.test.tsx`you can see other tests on components. Most are snapshot tests, which I personally have mixed feelings about. But I leave up to you whether it makes more sense snapshot tests, unit tests, e2e test, all of them or something in between :smile: